Source/WebKit/WebProcess/GPU/graphics/RemoteImageBufferProxy.cpp

```diff
@@ -337,7 +337,12 @@ void RemoteImageBufferProxy::putPixelBuffer(const PixelBuffer& pixelBuffer, cons
     // This means that putPixelBuffer() is only called when resolutionScale() == 1.
     ASSERT(resolutionScale() == 1);
     backingStoreWillChange();
-    remoteRenderingBackendProxy->putPixelBufferForImageBuffer(m_renderingResourceIdentifier, pixelBuffer, srcRect, destPoint, destFormat);
+    send(Messages::RemoteImageBuffer::PutPixelBuffer(Ref { const_cast<PixelBuffer&>(pixelBuffer) }, srcRect.location(), srcRect.size(), destPoint, destFormat));
+    // Small putPixelBuffers are batched, large ones are not.
+    if (pixelBuffer.size().unclampedArea() > 50 * 51) {
```

Comment:
The "50 x 51" deserves to be declared with a `const`, and this seems like a bit of an odd place for a size threshold behavior.